src/auth/auth.service.ts

```diff
@@ -134,11 +134,13 @@ export class AuthService {
             maxAge: AT_MAX_AGE * 1000,
             httpOnly: true,
             secure: true,
+            sameSite: "none",
```

Comment:
```suggestion
            sameSite: "none", // TODO: remove this in future when we use actual domains for the dashboard, they will likely be same-site
```

Reply:
just want to add a note here so we don't forget
I imagine the dashboard could be something like dashboard.chingu.io, and backend would be api.chingu.io